marlowe-dashboard-client/src/MainFrame/Lenses.purs

```diff
@@ -0,0 +1,33 @@
+module MainFrame.Lenses
```

Comment:
In the Marlowe Playground we put the lenses in the `Component.Types` module. Any reason to have them separate? We can discuss in an upcoming frontend-ps meeting and unify decisions in all projects

Reply:
I have a general preference for more and smaller modules (ditto functions). So that things can fit comfortably inside my small brain... :) I recently moved the Plutus playground lenses out into their own modules. But I will bow to the general will, if there's a strong preference for putting them in with the types.

Reply:
personally I'm happy with either

Reply:
I have no particular preference, but I would like to have it consistently defined.